.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
+*.bak
```

Comment:
I'm not sure, at some point vim (I assume) created / didn't clear it's backup files